### ISSUE TITLE
Email address is case sensitive

### ISSUE THREAD

**Issue by harianus** [crownstone/CrownstoneApp#13]
When creating an account with a capital letter you are not able to log in with an email address with all lower case letters.

**Comment by AlexDM0** [comment]
That's odd. I thought I explicitly force the email adresses to lower case before submitting. 

Thanks for reporting! I'll look into it!

**Comment by harianus** [comment]
You are right, it is the other way around. Creating an lowercase email address and then trying to log in with a non lowercase email address.

**Comment by AlexDM0** [comment]
Good catch! I'll try to get a new version in before new year!

Cheers

**Comment by AlexDM0** [comment]
Hi,

I've tried to reproduce this but I can't seem to manage. Are you using an android or ios phone?

Regards

**Comment by harianus** [comment]
Using the accept _Sphere_ page after an invite. So it's not in the app but in the website itself. Looks like a minor thing to me.

**Comment by AlexDM0** [comment]
I see, this is a cloud issue then. We'll address this soon!
